dbt/adapters/bigquery/python_submissions.py

```diff
@@ -98,7 +100,7 @@ def _submit_dataproc_job(self) -> dataproc_v1.types.jobs.Job:
                 "job": job,
             }
         )
-        response = operation.result(retry=self.retry)
+        response = operation.result(polling=self.retry)
```

Comment:
Should we consider rename `self.retry` to some thing different like `self.poll_policy` or `self.get_result_policy`.

Reply:
good call, I think `result_polling_policy` is the most explicit